bool success
string message
int16 code
float32 mean_squared_error
float32 coeficient_of_determination
float32 latitude
float32 longitude
float32 altitude
float32 orientation
